### PR TITLE
Html forms

### PR DESCRIPTION
**Pull request by Eugenepark215**

### CODE REVIEW

**Comment by Eugenepark215** [comment]
Screen Shot 2022-08-23 at 10 19 22 AM

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, great, much better so far. It just needs some `<label>` elements for each input.

image

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, sweet, now each form control element (input, select, radio, textarea) just needs an id attribute to match its label's `for`

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <div>
+      <label for="email">Email</label>
+      <div>
+        <input id="email" type="text" name="email" size="21" maxlength="30" />
+      </div>
+    </div>
+    <div>
+      <label for="why">Why is coding cool?</label>
+      <div>
+        <select name="option" id="option">
```
Comment (thebearingedge) [inline html-forms/index.html:18]: The `for` and `id` attribute values will need to match for the label to be clickable.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      </div>
+    </div>
+    <div>
+      <p>
+        <label for="language" id="language">Favorite Langauge (choose one)</label>
+      </p>
+      <div>
+        <input type="radio" name="language" value="HTML"/><label for="HTML">HTML</label>
+        <input type="radio" name="language" value="CSS"/><label for="CSS">CSS</label>
+        <input type="radio" name="language" value="JavaScript"/><label for="JavaScript">JavaScript</label>
```
Comment (thebearingedge) [inline html-forms/index.html:33]: Each of your radio buttons will need an `id` attribute that matches its label's `for` value.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <p>
+      <label for="information" id="information">Would you like more information?</label>
+    </p>
+    <div>
+      <input type="checkbox" name="information" id="signmeup"/><label for="signmeup">Sign me up!</label>
+    </div>
+    <div>
+      <label for="comments" id="comments">Comments</label>
+      <div>
+        <textarea name="comments" cols="35" rows="8"></textarea>
```
Comment (thebearingedge) [inline html-forms/index.html:45]: Your `textarea` needs an `id` attribute with a value matching the label's `for` value.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Here is an example of setting up label `for` and input `id` to match. You'll know it's working when you can click on the label to activate the input.

https://codepen.io/thebearingedge/pen/ExEJPjz?editors=1000

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**